packages/application/src/app.ts

```diff
@@ -24,12 +25,14 @@ export type QuetzFrontEndPlugin<T> = IPlugin<QuetzFrontEnd, T>;
  * App is the main application class. It is instantiated once and shared.
  */
 export class App extends Application<Shell> {
+  private _info: JupyterLab.IInfo = JupyterLab.defaultInfo;
```

Comment:
I would not reuse `JupyterLab.IInfo` as is because for now I don't see the point of having the attribute other than `deferred` and `disabled`.